pkg/cri/resource-manager/policy/builtin/static-pools/stp-policy.go

```diff
@@ -135,13 +108,15 @@ func (stp *stp) Description() string {
 func (stp *stp) Start(add []cache.Container, del []cache.Container) error {
 	var err error
 
-	if stp.conf == nil {
-		return stpError("cannot start without any configuration")
+	if err := stp.nodeUpdater.start(); err != nil {
```

Comment:
nitpick: you don't need a new error instance here.

Reply:
Changed now (in #478) so that the pre-defined err is removed and we only use combined assign-condition if statements